api/tests/test_game_state_server.py

```diff
@@ -282,10 +283,8 @@ async def test_delete_with_full_token(gss_with_client):
 
 @pytest.mark.asyncio
 async def test_room_full(gss_with_client):
-    for i in range(MAX_USERS_PER_ROOM):
+    for i in range(MAX_USERS_PER_ROOM - 1):
```

Comment:
I think that instead of this, we should change the check in game_state_server (line 45) to be <= MAX_USERS_PER_ROOM. That way, the max users per room will be the same as MAX_USERS_PER_ROOM instead of MAX_USERS_PER_ROOM - 1

Reply:
Good thought. This is done